generate/src/contract.rs

```diff
@@ -71,10 +71,12 @@ pub(crate) fn expand_contract(args: &Args) -> Result<TokenStream> {
 
     Ok(quote! {
         #doc
+        #[allow(warnings)]
```

Comment:
What are the warnings you are running into? Would you still be able to see them if you e.g. make changes the macro code?

Reply:
- unused functions (all contract functions are generated regardless of whether or not they are used).
- extra parenthesis (this happens because we generate code that works when there are parameters and when there are not).

Now that you mention it.... maybe we should be more restrictive with our warning suppression to help catch issues that otherwise would have been missed.

Note that these warnings only show up when using the generator API as warnings are not enabled at all from `proc_macro` generated code.

Reply:
Alright, we now have less catch all `allow` attributes so we can use compiler warnings to catch code booboos.